docs/spec/distribution/tags.md

```diff
@@ -0,0 +1,30 @@
+# Tags
+
+The distribution module emits the following events/tags:
+
+## Handlers
+
+### MsgSetWithdrawAddress
+
+| Key       | Value                     |
+|-----------|---------------------------|
+| delegator | {delegatorAccountAddress} |
```

Comment:
Is there a reason why we don't tag every message with it's type? Or do we and that isn't mentioned here?

Reply:
We do:

```
		tags = append(tags, sdk.MakeTag(sdk.TagAction, []byte(msg.Type())))
```

I will be changing this in #3171 to be `message-type` instead of `action` so I'd rather do that in the aforementioned PR.